charts/prefect-worker/templates/configmap.yaml

```diff
@@ -1,14 +1,14 @@
-{{- if .Values.worker.config.baseJobTemplate }}
+{{- if and .Values.worker.config.baseJobTemplate.enabled .Values.worker.config.baseJobTemplate.data }}
```

Comment:
if the user has `enabled` a base job template AND passed the JSON data, this config map will be created